src/plugins/vis_type_timeseries/server/plugin.ts

```diff
@@ -103,12 +99,7 @@ export class VisTypeTimeseriesPlugin implements Plugin<VisTypeTimeseriesSetup> {
     };
 
     (async () => {
```

Comment:
From my point of view, now we don't need to keep this extra wrapper` (async () => {.` I think we can safely remove it